content/en/docs/tasks/observability/distributed-tracing/skywalking/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+source "tests/util/addons.sh"
+
+# @setup profile=demo
+
+## Setting up application
+# Set to known setting of sidecar injection
+kubectl label namespace default istio-injection=enabled --overwrite
+
+_deploy_and_wait_for_addons skywalking
+
+# Install Bookinfo application
+startup_bookinfo_sample
+
+snip_accessing_the_dashboard_1 &
+
+_set_ingress_environment_variables
+GATEWAY_URL="$INGRESS_HOST:$INGRESS_PORT"
+bpsnip_trace_generation__1
+
+# Although test says, take a look at traces, we don't have to do that in this task
+# as it is covered by an integration test in istio/istio
+function access_skywalking_with_portforward() {
+  local skywalking_url='http://localhost:8080'
+  curl -s -o /dev/null -w "%{http_code}" "$skywalking_url"
```

Comment:
this only check skywalking UI is ok, we can port forward oap to localhost then do a query like:
```shell
function access_zipkin_with_portforward() {
  local zipkin_url='http://localhost:9411/zipkin/api/v2/traces?serviceName=productpage.default'
  curl -s -o /dev/null -w "%{http_code}" "$zipkin_url"
}
```
WDYT?

Reply:
I added a query to verify the UI and backend OAP service is OK but didn't verify the response

Reply:
I'm not that familiar with skywalking API, Is `reponse.code == 200` means oap receive real tracing data?